client/layout/guided-tours/tours/jetpack-monitoring-tour.js

```diff
@@ -67,7 +69,7 @@ export const JetpackMonitoringTour = makeTour(
 						) }
 					</p>
 					<ButtonRow>
-						<SiteLink isButton href={ '/checklist/:site' }>
+						<SiteLink isButton href={ '/plans/my-plan/:site' }>
```

Comment:
We could turn the `{ '...' }`s into `"..."`s while we're here (also below). Probably my bad :sweat_smile: